src/components/auth/AuthenticationOutage.js

```diff
@@ -15,20 +13,15 @@ export const useDetectAuthenticationOutage = () => {
 };
 
 const AuthenticationOutage = ({loginOption}) => (
-  <div className={getBEMClassName('alert', ['error'])}>
-    <span className={getBEMClassName('alert__icon', ['wide'])}>
-      <FAIcon icon="exclamation-circle" />
-    </span>
-    <Body>
-      <FormattedMessage
-        description="Authentication outage message"
-        defaultMessage={`
+  <ErrorMessage>
```

Comment:
Self-reminder: deal with 'wide icon'

Reply:
Before:
![Screenshot from 2023-11-06 11-52-16](https://github.com/open-formulieren/open-forms-sdk/assets/19154114/8fa8c1b6-eef9-46d6-9073-cd8a30f06cab)
After:
![Screenshot from 2023-11-06 11-53-09](https://github.com/open-formulieren/open-forms-sdk/assets/19154114/f6dc0a52-36ce-48a4-9aa4-78aaa708c8f6)
I dont think there is a need to make the icon wider? :thinking:

Reply:
looks fine to me